flink-end-to-end-tests/run-nightly-tests.sh

```diff
@@ -154,8 +154,8 @@ run_test "Test PubSub connector with Docker based Google PubSub Emulator" "$END_
 
 run_test "Avro Confluent Schema Registry nightly end-to-end test" "$END_TO_END_DIR/test-scripts/test_confluent_schema_registry.sh"
 
-run_test "State TTL Heap backend end-to-end test" "$END_TO_END_DIR/test-scripts/test_stream_state_ttl.sh file"
-run_test "State TTL RocksDb backend end-to-end test" "$END_TO_END_DIR/test-scripts/test_stream_state_ttl.sh rocks"
+run_test "State TTL Heap backend end-to-end test" "$END_TO_END_DIR/test-scripts/test_stream_state_ttl.sh file" "skip_check_exceptions"
+run_test "State TTL RocksDb backend end-to-end test" "$END_TO_END_DIR/test-scripts/test_stream_state_ttl.sh rocks" "skip_check_exceptions"
```

Comment:
I think there are still some other files which call this end-to-end test. For example `split_misc_hadoopfree.sh`, `split_misc.sh` are both calling this test.

Reply:
Ah, I just grepped the "test_stream_state_ttl.sh" in `flink-end-to-end` module, add the others